app/js/patternClas.js

```diff
@@ -0,0 +1,59 @@
+/*exported patternC */
+
+function patternC() {
+
+  "use strict";
+
+  function Plant(arg) {
+    this.pattern = "pseudoclassical";
+    this.species = arg.name;
+    this.price = arg.price || "brak";
+    this.lifeCycle = arg.lifeCycle || "-";
+    this.flowers = arg.flowers || "-";
+  }
+
+  Plant.prototype.getPrice = function () {
+    return this.price;
+  };
+
+  /**************** Tree ***************/
+
+  function Tree(arg) {
```

Comment:
You need some kind of error handling as well, a tree without options is probably not a tree at all.